Mailer/app/presenters/templates/@layout.latte

```diff
@@ -205,5 +205,6 @@
         {/foreach}
     });
 </script>
+{block javascript}{/block}
```

Comment:
Let's use `scripts` as a block label. we already use it in CRM; so we'd stay consistent here.